nutkit/frontend/driver.py

```diff
@@ -101,13 +101,15 @@ def send_and_receive(self, req, timeout=None, hooks=None, *,
         return self.receive(timeout=timeout, hooks=hooks,
                             allow_resolution=allow_resolution)
 
-    def execute_query(self, cypher, params=None,
-                      routing=None, database=None,
-                      impersonated_user=None, bookmark_manager=...):
+    def execute_query(self, cypher, *, params=None, routing=None,
```

Comment:
Params should be a positional argument or tests which uses it as positional should be updated.

Reply:
Of course :facepalm: 

Fixed it!